spec/unit/prepare.spec.js

```diff
@@ -833,4 +834,78 @@ describe('prepare', () => {
             ).toBeResolved();
         });
     });
+
+    describe('updateProjectAccordingTo', function () {
+    describe('relocate MainActivity.java', function () {
```

Comment:
sub describe blocks should have an indentation level. Functions should be in an arrow function format.

```suggestion
    describe('updateProjectAccordingTo', () => {
    describe('relocate MainActivity.java', () => {
```

Reply:
This was a consequence of basing this test on an existing test